lib/gollum-lib/macro/flash.rb

```diff
@@ -2,7 +2,7 @@ module Gollum
   class Macro
     class Flash < Gollum::Macro
       def render(message, type='')
-        flash_type << "flash-#{type}" if ['warn', 'error', 'success'].include?(type)
+        flash_type = "flash-#{type}" if ['warn', 'error', 'success'].include?(type)
```

Comment:
I was wondering about this! What if `type` is not one of these valid values, though? Won't the string interpolation on the next line throw an error because the variable is unknown?

Reply:
That's what I would have expected too, but in testing no such error surfaced. I've changed the code regardless to set a default value.